mmcv/utils/logging.py

```diff
@@ -28,6 +28,12 @@ def get_logger(name, log_file=None, log_level=logging.INFO, file_mode='w'):
         logging.Logger: The expected logger.
     """
     logger = logging.getLogger(name)
+
+    # set the root logger's StreamHandler, if any, to log at the ERROR level
+    for handler in logger.root.handlers:
```

Comment:
should those lines be executed after Line 37?

```python
if name in logger_initialized:
    return logger
 
for handler in logger.root.handlers:
   ...
```

Reply:
More comments should be added here.

Reply:
Great suggestions! I've modified the code and added more comments accordingly.